src/fortran/fct_interface.f90

```diff
@@ -269,6 +274,16 @@ integer function th_getpartialmolarintenergies_r(n, ie)
         double precision, intent(out) :: ie(*)
     end function th_getpartialmolarintenergies_r
 
+    integer function th_getpartialmolarcp(n, cpbar)
+      integer, intent(in) :: n
+      double precision, intent(out) :: cpbar(*)
+    end function th_getpartialmolarcp
```

Comment:
Please match the indentation style of the surrounding code (4 spaces per level).